src/Appwrite/Platform/Modules/Functions/Workers/Builds.php

```diff
@@ -0,0 +1,850 @@
+<?php
+
+namespace Appwrite\Platform\Modules\Functions\Workers;
+
+use Ahc\Jwt\JWT;
+use Appwrite\Event\Event;
+use Appwrite\Event\Func;
+use Appwrite\Event\Usage;
+use Appwrite\Messaging\Adapter\Realtime;
+use Appwrite\Utopia\Response\Model\Deployment;
+use Appwrite\Vcs\Comment;
+use Exception;
+use Executor\Executor;
+use Swoole\Coroutine as Co;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Config\Config;
+use Utopia\Database\Database;
+use Utopia\Database\DateTime;
+use Utopia\Database\Document;
+use Utopia\Database\Exception\Conflict;
+use Utopia\Database\Exception\Restricted;
+use Utopia\Database\Exception\Structure;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Validator\Authorization;
+use Utopia\Logger\Log;
+use Utopia\Platform\Action;
+use Utopia\Queue\Message;
+use Utopia\Storage\Device;
+use Utopia\Storage\Device\Local;
+use Utopia\System\System;
+use Utopia\VCS\Adapter\Git\GitHub;
+
+class Builds extends Action
+{
+    public static function getName(): string
+    {
+        return 'builds';
```

Comment:
Shouldn't it be `buildFunctions`? And it is common for both functions and sites, then we need to move it out of Functions directory.